### PR TITLE
Update README.md to be more colorblind friendly

### PR DESCRIPTION
**Pull request by jatins**
Checklist before merging this PR:
- [x] Mentioned all issues that this PR fixes or addresses.
- [x] Summarized the updates of this PR under **Summary**.
- [x] Added an entry under **Unreleased** in the [Changelog](../CHANGELOG.md).

Fixes #.

### Summary
I had trouble differentiation green/red squares in readme, this make it a little more colorblind friendly. I just find/replaced the emojis so the rest of text should be unchanged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,55 +218,55 @@ Here's a breakdown of the forecasting models currently implemented in Darts. We
 on bringing more models and features.
 
 
+
```
Comment (dennisbader) [inline README.md:221]:
```suggestion
```

**Review by dennisbader — CHANGES_REQUESTED** [review]
Thanks for this PR @jatins, I think that makes sense :)
Could you add an entry to the changelog? Then we're ready to merge 🚀

**Comment by jatins** [comment]
@dennisbader thanks for having a look! Changes:
* Updated the changelog 
* removed redundant whitespace

**Review by dennisbader — APPROVED** [review]
Thanks a lot @jatins, ready to merge 🚀

**Pull request merged**